OurUmbraco/Auth/ProjectAuthConstants.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace OurUmbraco.Auth
+{
+    public class ProjectAuthConstants
+    {
+        public const string ProjectIdClaim = "http://our.umbraco/projects/projectid";
+        public const string MemberIdClaim = "http://our.umbraco/projects/memberid";
+        public const string BearerTokenClaimType = "http://our.umbraco/projects";
```

Comment:
These are not valid URLs? And not `https` - maybe the URL prefix should be configurable as well (so a global variable for the URL `https://our.umbraco.com/`. It has changes from `.org` to `.com` before and maybe in the future we'd want to move this to  `packages.umbraco.com` (I foresee we really might if we end up splitting up Our a bit more).

Reply:
Seems it just needs to be a unique string value..
https://docs.microsoft.com/en-us/dotnet/framework/wcf/extending/how-to-create-a-custom-claim